packages/node_modules/@webex/react-component-space-item/src/index.js

```diff
@@ -4,11 +4,13 @@ import PropTypes from 'prop-types';
 import {Avatar, Button, SpaceListItem} from '@collab-ui/react';
 import Timer from '@ciscospark/react-component-timer';
 import {getTeamColor} from '@ciscospark/react-component-utils';
+import * as NotificationsConstants from '@ciscospark/react-component-utils/src/constants';
```

Comment:
You shouldn't need the `/src/constants`. If you want a group of constants, export the constants as a group from `react-component-utils/index.js`